src/pages/[platform]/build-a-backend/auth/add-social-provider/index.mdx

```diff
@@ -407,7 +407,99 @@ After configuring the OAuth endpoints (with Cognito Hosted UI), you can integrat
 
 A custom state is not required, but if you are looking to add one, you are able to do so by passing a string value (e.g. `signInWithRedirect({ customState: 'xyz' })`) and listening for the custom state via [Hub](/[platform]/build-a-backend/utilities/hub/). You can also use the Hub eventing system to catch errors using `signInWithRedirect()` to listen for the `signInWithRedirect_failure` event.
 
-<InlineFilter filters={["javascript"]}>
+<InlineFilter filters={["javascript", "angular", "vue"]}>
```

Comment:
Does this include all the relevant platforms? "react", "nextjs" are not here rn.

Reply:
+1

Reply:
yes, is included below

Reply:
ohok thank you so much @elorzafe @elorzafe